pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -157,13 +157,17 @@ func (rc *reconciler) isMultiAttachForbidden(volumeSpec *volume.Spec) bool {
 			return false
 		}
 
+		seenMany := false
 		// check if this volume is allowed to be attached to multiple PODs/nodes, if yes, return false
 		for _, accessMode := range volumeSpec.PersistentVolume.Spec.AccessModes {
+			if accessMode == v1.ReadWriteOnce {
```

Comment:
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
volume can have multiple access mode. Having readwriteonce does not mean that multi-attach is forbidden.

Reply:
How about adding a test for the mixed access mode case ?

Reply:
you mean adding test cases for this change? But I think we cannot make this change based on my understanding the access modes. We might need to clarify the meaning of multiple access code in the doc.

Reply:
I mean adding test case which shows this change would fail the new case.

Reply:
could you give me an example?